pkgs/tools/text/enscript/default.nix

```
@@ -9,7 +9,13 @@ stdenv.mkDerivation rec {
     sha256 = "1fy0ymvzrrvs889zanxcaxjfcxarm2d3k43c9frmbl1ld7dblmkd";
   };
 
-  preBuild =
+  patches = [
+    # fix compile failure on macos. use system getopt like linux
+    # https://savannah.gnu.org/bugs/?64307
+    ./0001-use-system-getopt.patch
+  ];
+
+  postPatch =
```

```suggestion
  postPatch = ''
```
That should be one line up